app/components/post_draft/send_action/index.tsx

```diff
@@ -67,15 +103,32 @@ function SendButton({
             disabled={disabled}
             onLongPress={showScheduledPostOptions}
         >
-            <View style={viewStyle}>
-                <CompassIcon
-                    name='send'
-                    size={24}
-                    color={buttonColor}
-                />
-            </View>
+            <Tooltip
+                isVisible={scheduledPostTooltipVisible}
+                useInteractionManager={true}
+                placement='top'
+                content={<ScheduledPostTooltip onClose={onCloseScheduledPostTooltip}/>}
+                onClose={onCloseScheduledPostTooltip}
+                tooltipStyle={style.scheduledPostTooltipStyle}
+            >
+                <View style={viewStyle}>
+                    <CompassIcon
+                        name='send'
+                        size={24}
+                        color={buttonColor}
+                    />
+                </View>
+            </Tooltip>
         </TouchableWithFeedback>
     );
 }
 
-export default SendButton;
+const enhanced = withObservables([], () => {
+    const scheduledPostFeatureTooltipWatched = observeTutorialWatched(Tutorial.SCHEDULED_POST);
+
+    return {
+        scheduledPostFeatureTooltipWatched,
+    };
+});
+
+export default withDatabase(enhanced((SendButton)));
```

Comment:
index should only contain this and then we should move the rest to a `send_action.tsx` file

Reply:
Done.